nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/web/dao/impl/StandardProcessorDAO.java

```diff
@@ -382,6 +382,7 @@ private void verifyUpdate(ProcessorNode processor, ProcessorDTO processorDTO) {
                     configDTO.getSchedulingPeriod(),
                     configDTO.getSchedulingStrategy(),
                     configDTO.getExecutionNode(),
+                    configDTO.isExecutionNodeRestricted(),
```

Comment:
This should not be checked, as it's not a configuration element that the user is able to configure.